src/Viewer/ViewerGUI.hpp

```diff
@@ -1,6 +1,5 @@
 /***************************************************************************
  * @file   ViewerGUI.hpp
- * @brief  Generates a Graphical User Interface for the Viewer application.
  *
  * @authors Tobias Bertel, Mingze Yuan, Reuben Lindroos, Christian Richardt
  ***************************************************************************/
```

Comment:
We can probably delete this header as we don't need it.

Reply:
https://github.com/cr333/OmniPhotos/blob/faba49c23d4aad75f4a85e47618ff4862e007746/src/Viewer/ViewerApp.hpp#L17